src/index.html

```diff
@@ -10,13 +10,102 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>Moyo Title</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="MOYO Header Image"
+        />
+      </a>
+      <nav class="nav-container">
+        <ul class="nav_links">
+          <li>
+            <a
+              href="#section-1"
+              class="section-1 nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#section-2"
+              class="section-2 nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#section-3"
+              class="section-3 nav_link"
```

Comment:
Why do you need these classes `section-3`, `section-4`...
They are not really needed